.rubocop.yml

```diff
@@ -4,7 +4,9 @@ Documentation:
   Enabled: false
 AllCops:
   Exclude:
-    - 'bin/**/*'    
+    - 'bin/**/*'
+    - 'db/migrate/*'
```

Comment:
What's the reason you disabled all cops for migrations?

Reply:
because you said it when we were setuping rubocop

Reply:
😯